sesman/startwm.sh

```diff
@@ -55,17 +55,38 @@ post_start()
   return 0
 }
 
+get_xdg_session_startupcmd()
+{
+  # DESKTOP_SESSION should be set in sesman.ini in the SessionVariables section.
+  # If set and valid then the STARTUP command will be taken from there
+  if [ -n "$1" ] && [ -d /usr/share/xsessions ] \
+    && [ -f "/usr/share/xsessions/$1.desktop" ]; then
+    STARTUP=$(grep ^Exec= "/usr/share/xsessions/$1.desktop")
+    STARTUP=${STARTUP#Exec=*}
+    XDG_CURRENT_DESKTOP=$(grep ^DesktopNames= "/usr/share/xsessions/$1.desktop")
+    XDG_CURRENT_DESKTOP=${XDG_CURRENT_DESKTOP#DesktopNames=*}
+    echo "env XDG_CURRENT_DESKTOP=$XDG_CURRENT_DESKTOP $STARTUP"
```

Comment:
Suggest `echo "env XDG_CURRENT_DESKTOP=\"$XDG_CURRENT_DESKTOP \" $STARTUP"`, to add "" to the XDG_CURRENT_DESKTOP

Reply:
true